<?xml version="1.0" ?>
<launch>
    <param name="/use_sim_time" value="true" />

    <!-- base_footprint and base_link was missing -->
    <node pkg="tf" type="static_transform_publisher" name="odom_T" args="0.0 0.0 0.0102 0.0 0.0 0.0 1.0 /base_footprint /base_link 100"/>

     <group ns="cslibs_mapping_node_2d">
     <rosparam param="laser_topics">
         ["/scan"]
     </rosparam>
     <rosparam param="laser_queue_sizes">
         [10]
     </rosparam>
     <param name="subscriber_queue_size" value="10" />
     <param name="rate" value="0.0" />
     <param name="resolution" value="0.05" />
     <param name="map_frame" value="/odom" />
     <param name="ndt_map_pub_rate" value="10.0" />
     <param name="occ_map_pub_rate" value="10.0" />
     <param name="ndt_sampling_resolution" value="0.025" />
     <param name="ndt_grid_resolution" value="0.5" />
     <param name="rate" value="60.0" />
    </group>

    <node name="cslibs_mapping_node_2d" pkg="cslibs_mapping" type="cslibs_mapping_node_2d" output="screen" />

</launch>
